container/nginx/confs/proxy.tlsconf

```diff
@@ -20,6 +20,7 @@ map $format_query $format_final {
 
 server {
     listen 443 default_server ssl http2 reuseport;
+    listen 443 http3 reuseport;
```

Comment:
I might be wrong, but isn't `reuseport` only used once per address/port combination (ie we can use it on both the listen directives for port 443, only one) ? 

[Source for this:](http://nginx.org/en/docs/http/ngx_http_core_module.html#listen) 

> The listen directive can have several additional parameters specific to socket-related system calls. These parameters can be specified in any listen directive, but only once for a given address:port pair.

Reply:
why not put `http3` in line 22? I'm not sure if two listen directives can have the same port for the same IPv4 interface.

Reply:
good point @AmeanAsad!

@DiegoRBaquero I initially did that, but I got `nginx: [emerg] "listen" directive "ssl" parameter is incompatible with "http3" in /etc/nginx/conf.d/proxy.tlsconf:22`.
Taking https://quic.nginx.org/readme.html, as an example, I'm thinking this section should be:

```
listen 443 default_server ssl http2;
listen 443 http3 reuseport;
```

Reply:
If the parameters are shared for all listen directives with same `address:port` then we should set `reuseport` in the first one (I don't know if it matters, just rather waterfall than only activated for http3, as CPU would probably spike BAD without it in http2)

thinking of:
```
listen 443 default_server ssl http2 reuseport;
listen 443 http3;
```

Reply:
Alright let's go with that just in case